data/routing.txt

```diff
@@ -15,8 +14,15 @@ Bitcoin         | Coinbase   | 1% + 15¢     | On request |
 
 Route           | Provider   | Fee          | When       | Details
 ----------------+------------+--------------+------------+-----------
-Bank Account    | Balanced   | 0            | At payday  | U.S. only
```

Comment:
> Current target is June 18 for final ACH run on Balanced, and June 25 for initial run on Zipmark.

So, the `In` route was not through the ACH, right? When it was decommissioned?

The graph becomes more complicated - need to make clear that Credit Card belongs to Bank and Bank and Credit Card didn't change during the transition. Then the money from the Bank are being transferred by Balanced or Braintree (how to call them?) for the `In` direction.

And then money go into the Bank, they move first to ACH by Zipmark and then to Card of the Bank. Is that right?